packages/rollup-contracts/contracts/StateManager.sol

```diff
@@ -7,17 +7,17 @@ pragma solidity ^0.5.0;
  */
 contract StateManager {
     // Storage
-    function getStorage(address _ovmContractAddress, bytes32 _slot) internal view returns(bytes32);
-    function setStorage(address _ovmContractAddress, bytes32 _slot, bytes32 _value) internal;
+    function getStorage(address _ovmContractAddress, bytes32 _slot) public view returns(bytes32);
```

Comment:
Same comment as above on all of these methods WRT making `external` when possible.